lib/local-storage.coffee

```diff
@@ -14,14 +14,12 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ###
 
-settings = require('resin-settings-client')
-
-# Fallback to filesystem based
-# storage if not in the browser.
-unless localStorage?
-	LocalStorage = require('node-localstorage').LocalStorage
-
+if typeof localStorage isnt 'undefined'
```

Comment:
I personally prefer the question mark once since its more concise, and doesn't compare to a string (at least from the point of view of the programmer), which could lead to typo errors, etc.